src/homelogstore/log_store_family.cpp

```diff
@@ -300,23 +300,23 @@ nlohmann::json LogStoreFamily::dump_log_store(const log_dump_req& dump_req) {
     return json_dump;
 }
 
-nlohmann::json LogStoreFamily::get_status(const int verbosity) const {
-    nlohmann::json js;
+sisl::status_response LogStoreFamily::get_status(const sisl::status_request& request) const {
+    sisl::status_response response;
     auto unopened = nlohmann::json::array();
     for (const auto& l : m_unopened_store_id) {
         unopened.push_back(l);
     }
-    js["logstores_unopened"] = std::move(unopened);
+    response.json["logstores_unopened"] = std::move(unopened);
 
     // Logdev status
-    m_log_dev.get_status(verbosity, js);
+    response.json["log_dev"] = m_log_dev.get_status(request).json;
 
     // All logstores
     m_id_logstore_map.withRLock([&](auto& id_logstore_map) {
         for (const auto& [id, lstore] : id_logstore_map) {
```

Comment:
@sanebay @hkadayam 
Does it make sense that there exists a range parameter in `status_request` to bound the status of a subset of the children? 
Generally speaking, the parameter can be `std::string extra_request_param` and each object can interpret it in their own way. Here it can interpreted as a range.

Reply:
It has been already designed and implemented as `status_request.json`